benchmarks/analyzeNode/analyze_node_test.go

```diff
@@ -0,0 +1,33 @@
+package detectconent
+
+import (
+	"fmt"
+	"runtime"
+	"testing"
+
+	node "github.com/stackrox/scanner/pkg/analyzer/nodes"
+	"github.com/stretchr/testify/require"
+
+	// Register the Docker image extractor
+	_ "github.com/stackrox/scanner/ext/imagefmt/docker"
+)
+
+func BenchmarkAnalyzeNode(b *testing.B) {
+	runBenchmarkAnalyzeNode(b, "fake path")
+}
+
+func runBenchmarkAnalyzeNode(b *testing.B, pathName string) {
+	var m1, m2 runtime.MemStats
+	b.ResetTimer()
+	runtime.GC()
+	runtime.ReadMemStats(&m1)
+
+	for i := 0; i < b.N; i++ {
+		var err error
+		_, err = node.Analyze("testNode", pathName, node.AnalyzeOpts{UncertifiedRHEL: false, IsRHCOSRequired: true})
+		require.NoError(b, err)
+	}
+	runtime.ReadMemStats(&m2)
```

Comment:
The way we gather memory stats may be inaccurate - we measure memory before the benchmark and after it. In between, there will be more that one execution of `Analyze`. I think, we should report memory allocation per single call to Analyze...

or skip doing this this way and rely on the numbers that go reports automatically with benchmem: ` go test -bench=foo -benchmem`.

I would be in favor of the second option.